src/main/kotlin/app/revanced/patches/messenger/inputfield/fingerprints/SendPresenceTypingIndicatorFingerprint.kt

```diff
@@ -0,0 +1,29 @@
+package app.revanced.patches.messenger.inputfield.fingerprints
+
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import org.jf.dexlib2.Opcode
+
+object SendPresenceTypingIndicatorFingerprint : MethodFingerprint(
```

Comment:
Is the fingeprint certainly gonna work for future versions of the app?

Reply:
Is there a way to match a field of the class that defines a function? There is a very distinct string in the class that contains this function, it would be great if that can be accessed from `customFingerprint`

Reply:
No, but the customFingerprint function can be modified in ReVanced Patcher so that it consumes more parameters such as the current classDef.